### PR TITLE
fix: verify if addressIP exists

### PR DESCRIPTION
**Pull request by PascalBourdier**
#### What type of PR is this?

/kind bug
#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: cmd/kubeadm/app/util/config/initconfiguration.go [hunk 1]
```diff
@@ -142,7 +142,7 @@ func SetAPIEndpointDynamicDefaults(cfg *kubeadmapi.APIEndpoint) error {
 
 	// kubeadm allows users to specify address=Loopback as a selector for global unicast IP address that can be found on loopback interface.
 	// e.g. This is required for network setups where default routes are present, but network interfaces use only link-local addresses (e.g. as described in RFC5549).
-	if addressIP.IsLoopback() {
+	if addressIP != nil && addressIP.IsLoopback() {
```
Comment (neolit123) [inline cmd/kubeadm/app/util/config/initconfiguration.go:145]: has this been panicing under some conditions?

please include details in the PR description, perhaps also reproduction steps (given there is no linked issue).

Comment (PascalBourdier) [reply cmd/kubeadm/app/util/config/initconfiguration.go:145]: No use case but I just found a potential bug here

Comment (aojea) [reply cmd/kubeadm/app/util/config/initconfiguration.go:145]: Line 139 already validates this potential bug

Comment (neolit123) [reply cmd/kubeadm/app/util/config/initconfiguration.go:145]: yes, for "" as input, nil will always be returned.
@PascalBourdier please send other fixes, this is not needed.
/close

**Pull request closed without merging**